### DISCUSSION TITLE
Additional PHP CodeGen

### DISCUSSION THREAD

**Discussion by nickpoulos** [usebruno/bruno#2591] [Ideas]
I would love to see some additional code generation for PHP using different clients.  The big two would be Guzzle and the Laravel Http Client.  

I was trying to look through the repo to make a PR and was able to find the location of the CodeGen menu and the available options. But I was not quite able to find where the code generation was actually handled.  

If somebody can point me in the right direction, I can give it the PR a try myself. Otherwise I am happy to provide some solid examples for both clients that somebody more talented can integrate into Bruno.  

Thanks for the great tool.

**Comment by Its-treason** [comment]
The code generation itself is handled by: https://github.com/Kong/httpsnippet I think they already support PHP with Guzzle but not with Laravel.


The library is then used here: https://github.com/usebruno/bruno/blob/main/packages%2Fbruno-app%2Fsrc%2Fcomponents%2FSidebar%2FCollections%2FCollection%2FCollectionItem%2FGenerateCodeItem%2Findex.js
